app/services/sufia/workflow/workflow_by_admin_set_strategy.rb

```diff
@@ -0,0 +1,16 @@
+module Sufia
+  module Workflow
+    class WorkflowByAdminSetStrategy
+      attr_reader :admin_set_id
+      def initialize(work, attributes)
+        @work = work
+        @admin_set_id = attributes[:admin_set_id]
+      end
+
+      # @return [String] The name of the workflow by admin_set to use
+      def workflow_name
+        @admin_set_id ||= 'default'
```

Comment:
Sorry to do this, but I now think my original advice here was bad. I suspect `workflow_name` should just return `@admin_set_id`, and the assignment to the ivar in `initialize` should probably look like `@admin_set_id = attributes.fetch(:admin_set_id, 'default')`